spec/features/events_spec.rb

```diff
@@ -79,8 +79,27 @@
       FactoryGirl.create(:application_letter_accepted, :event => @event, :user => @pupil.user)
     end
     visit event_path(@event)
-    click_button I18n.t('events.applicants_overview.sending_acceptances')
-    expect(page).to have_selector('div', :id => 'send-emails-modal')
+    click_link I18n.t('events.applicants_overview.sending_acceptances')
+    fill_in('email_subject', with: 'Subject')
+    fill_in('email_content', with: 'Content')
+    click_button I18n.t('emails.email_form.send')
+    expect(ActionMailer::Base.deliveries.last.to.count).to eq(1)
```

Comment:
with this line you only assure that the last email was adressed to exactly one person, but not that an email was sent